### PR TITLE
autoload is not available with addPlugin

### PR DESCRIPTION
**Pull request by saeideng**

### CODE REVIEW

File: en/plugins.rst [hunk 1]
```diff
@@ -94,11 +94,15 @@ Additionally, you will need to tell Composer to refresh its autoloading cache:
...
 autoloading with ``Plugin``::
 
-    // In App\Application::bootstrap() add.
-    $this->addPlugin('ContactManager', ['autoload' => true]);
+    Plugin::load('ContactManager', ['autoload' => true]);
 
-    // Prior to 3.6.0 you need to use Plugin::load()
 
+.. deprecated:: 3.7.0
+    Plugin::load() and `autoload` option  are deprecated.
```
Comment (dereuromark) [inline en/plugins.rst]: do we need double ` here?

Comment (saeideng) [reply en/plugins.rst]: I dont know enough about formatting on docs :(

Comment (dereuromark) [reply en/plugins.rst]: See the above text :)

Comment (saeideng) [reply en/plugins.rst]: I did not  run it in local

Comment (dereuromark) [reply en/plugins.rst]: no need to.

Comment (saeideng) [reply en/plugins.rst]: this is within a box (deprecated box) I'm not sure really about affect it on ``  :)
if you are sure I can change it

File: en/plugins.rst [hunk 2]
```diff
@@ -94,11 +94,15 @@ Additionally, you will need to tell Composer to refresh its autoloading cache:
...
-    $this->addPlugin('ContactManager', ['autoload' => true]);
+    Plugin::load('ContactManager', ['autoload' => true]);
 
-    // Prior to 3.6.0 you need to use Plugin::load()
 
+.. deprecated:: 3.7.0
+    Plugin::load() and ``autoload`` option  are deprecated.
+
+.. note::
+    IMPORTANT: ``autoload`` option is not available on ``addPlugin()``, you should use composer dumpautoload instead.
```
Comment (dereuromark) [inline en/plugins.rst]: `composer dumpautoload` might also need quoting as code.

**Pull request merged**